homeworks/vadym.myroshnyk_VadimMir/homework_7/homework_7.js

```diff
@@ -0,0 +1,19 @@
+// fibonacci
+const randomNum = Math.floor(Math.random() * (100 - 50 + 1) + 50);
+
+function fib(max) {
+    const arr = [0];
+    let a = 1;
+    let b = 0;
+    for (let i = 0; i < max; i += a) {
```

Comment:
this check is quite confusing, why not just iterating with a `while` loop until sum of two last elements in array gets higher than `max` value